src/styles/mixins.scss

```diff
@@ -0,0 +1,13 @@
+@mixin rotate-animation($duration, $timing-function) {
+  animation: hand-move $duration $timing-function infinite;
+  transform-origin: bottom center;
```

Comment:
The `transform-origin` property should be part of the element's styling where the mixin is included rather than in the mixin itself to keep the mixin focused on the animation only.